src/index.ts

```diff
@@ -235,12 +236,14 @@ export const initializeKeypair = async (
     await addKeypairToEnvFile(keypair, envVariableName, envFileName);
   }
 
-  await airdropIfRequired(
-    connection,
-    keypair.publicKey,
-    airdropAmount || DEFAULT_AIRDROP_AMOUNT,
-    minimumBalance || DEFAULT_MINIMUM_BALANCE,
-  );
+  if (requestAirdropIfRequired) {
+    await airdropIfRequired(
+      connection,
+      keypair.publicKey,
+      airdropAmount || DEFAULT_AIRDROP_AMOUNT,
+      minimumBalance || DEFAULT_MINIMUM_BALANCE,
+    );
+  }
```

Comment:
I like this idea of speeding this up where we don't need to airdrop. 

We should be able to make this simpler by just allowing them to set `airdropAmount` to 0 or null. 

Then in the docs:

> If `airdropAmount` is specified, it will check the account's balance. If the balance is below the `minimumBalance`, it will airdrop the account `airdropAmount`.

Then here we can do: 

`if (airdropAmount) { ...`

Reply:
I was thinking the same method here, but decided to go with adding a boolean. I'm game for which ever as long as we can declare "I dont want the airdrop"

Reply:
Sweet. I'm teaching next week will Wed but I can either take care of this after or you can, happy either way.

Reply:
Updated to do so :)